enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -0,0 +1,310 @@
+---
+title: apply-user-defined-tags-to-all-azure-resources-created-by-openShift
+authors:
+  - "@bhb"
+reviewers:
+  - @patrickdillon
+  - @sdodson
+  - @jerpeter1
+  - @Miciah
+  - @sinnykumari
+  - @dmage
+  - @staebler
+  - @tkashem
+  - @tjungblu
+approvers:
+  - @sdodson
+  - @jerpeter1
+  - @bparees
+api-approvers:
+  - @joelspeed
+creation-date: 2022-07-12
+last-updated: 2022-07-12
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-8155
+see-also:
+  - "enhancements/api-review/custom-tags-aws.md"
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+
+# Apply user defined tags to all Azure resources created by OpenShift
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of Openshift to 
+have the ability to apply user defined tags to those resources created by Openshift 
+in Azure.
+
+## Motivation
+
+Motivations include but are not limited to:
+
+- Allow admin, compliance, and security teams to keep track of assets and objects 
+  created by OpenShift in Azure.
+
+### User Stories
+
+- As an openshift administrator, I want to have tags added to all resources created 
+  in Azure by Openshift.
+- As an openshift administrator, I want to restrict access granted to Openshift specific account.
+
+### Goals
+
+- The administrator or service (in the case of Managed OpenShift) installing OpenShift 
+  can configure allowed number of user-defined tags in the Openshift installer generated
+  install config, which is referred and applied by installer and the in-cluster operators
+  on the the Azure resources during creation.
+- Tags must be applied at creation time, in an atomic operation. It isn't acceptable 
+  to create an object and, after some period of time, apply the tags post-creation.
+
+### Non-Goals
+
+- Management of resource tags post creation of cluster is out of scope.
```

Comment:
I think the intent of this is the set of user tags can be set at cluster creation and are immutable, the tags are set on resources created by the cluster day-2.  Is that correct?

Reply:
Yes, correct, meant to state, updating or deleting tags added during cluster creation is not supported.